src/metabase/driver/postgres.clj

```diff
@@ -475,7 +486,9 @@
    (keyword "double precision")           :type/Float
    (keyword "time with time zone")        :type/Time
    (keyword "time without time zone")     :type/Time
-   (keyword "timestamp with timezone")    :type/DateTime
+   ;; TODO postgres also supports `timestamp(p) with time zone` where p is the precision
+   ;; maybe we should switch this to use `sql-jdbc.sync/pattern-based-database-type->base-type`
+   (keyword "timestamp with timezone")    :type/DateTimeWithTZ
```

Comment:
Can anyone gut-check this? any reason why we had this as `type/Datetime` before?